sdk/cosmos/azure-spring-data-cosmos/src/main/java/com/azure/spring/data/cosmos/config/AbstractCosmosConfiguration.java

```diff
@@ -26,12 +25,12 @@
     /**
      * Declare CosmosDbFactory bean.
      *
-     * @param config of cosmosDbFactory
+     * @param cosmosAsyncClient of cosmosDbFactory
      * @return CosmosDbFactory bean
      */
     @Bean
-    public CosmosFactory cosmosDBFactory(CosmosConfig config) {
-        return new CosmosFactory(config);
+    public CosmosFactory cosmosDBFactory(CosmosAsyncClient cosmosAsyncClient) {
```

Comment:
I guess I missed this when rebranding, but we should rename this API to -> `cosmosFactory()`
We don't want `cosmosDB`